src/containers/SocialLinksGroup.jsx

```diff
@@ -0,0 +1,25 @@
+import SocialLink from "../components/SocialLink";
+
+const SocialLinksGroup = () => {
+  return (
+    <div className="flex justify-center">
+      <SocialLink
+        icon="fa-brands fa-square-instagram"
+        href={"#"}
+        text={"Instagram"}
+      />
+      <SocialLink
+        icon="fa-brands fa-square-twitter"
+        href={"#"}
+        text={"Twitter"}
+      />
+      <SocialLink
+        icon="fa-brands fa-square-facebook"
+        href={"#"}
+        text={"Facebook"}
+      />
+    </div>
+  );
+};
+
+export default SocialLinksGroup;
```

Comment:
This code is much more scalable, easy to maintain and non-repetative.

Try to understand the concept in this code. Especially map and key

```

import SocialLink from "../components/SocialLink";

const socialLinksData = [
  {
    icon: 'fa-brands fa-square-instagram',
    href: '#',
    text: "Instagram",
  },
  {
    icon: 'fa-brands fa-square-facebook',
    href: '#',
    text: "Facebook",
  },
  {
    icon: 'fa-brands fa-square-twitter',
    href: '#',
    text: "Twitter",
  },
]

const SocialLinksGroup = () => {
  return (
    <div className="flex justify-center gap-2">
      {
        socialLinksData.map((socialData) => {
          return (
            <SocialLink
              key={socialData.text}
              icon={socialData.icon}
              href={socialData.href}
              text={socialData.text}
            />
          )
        })
      }
    </div>
  );
};

export default SocialLinksGroup;

```

Reply:
Implemented based on review comments